### ISSUE TITLE
Windows Deployment

### ISSUE THREAD

**Issue by tataDan** [MoAlyousef/fltk-rs#636]
I created a simple application on one Windows 10 computer and built it successfully using the MSVC toolchain.  It ran successfully.

I then copies the whole project file (Cargo.toml, src, target, etc.) to another Windows 10 computer that does not have the MSVC build tools installed on it.  When I attempt to run it I get the message that it can not run because VCRUNTIME140.dll is missing.

What am I doing wrong?

**Comment by MoAlyousef** [comment]
Are you using other crates with fltk?
FLTK doesn't link to the VC runtime since it doesn't use the C++ standard library.
Anyways, Windows 10 installations usually have the `Visual C++ Redistributable Packages` installed which contains system dll's.
You basically have 3 options to avoid such occurrences.
- Link the C runtime statically:
```toml
# .cargo/config.toml
[target.x86_64-pc-windows-msvc]
rustflags = ["-C", "target-feature=+crt-static"]

[target.i686-pc-windows-msvc]
rustflags = ["-C", "target-feature=+crt-static"]
```
- Deploy the  `Visual C++ Redistributable Packages` with your application, or just the headers.
- Build your app using the gnu toolchain.

**Comment by tataDan** [comment]
On the "development" computer I made the changes to .cargo\config.toml, but the application would not build until I changed:

`[target.x86_64-pc-windows-msvc]`

to

`[target.stable-x86_64-pc-windows-msvc]`.

However, I still got the same error on the "end-user" computer with the newly built application.
I discovered that vcruntime140.dll was missing from the computer so I downloaded and installed the Visual C++ Redistributable package. Now the application runs on both computers.
Thanks, once again, for you quick and helpful response.
